Source/Core/Core/CoreTiming.cpp

```diff
@@ -392,9 +405,10 @@ void Advance()
 {
 	MoveEvents();
 
-	int cyclesExecuted = slicelength - PowerPC::ppcState.downcount;
+	int cyclesExecuted = slicelength - DowncountToCycles(PowerPC::ppcState.downcount);
 	globalTimer += cyclesExecuted;
-	PowerPC::ppcState.downcount = slicelength;
+	lastOCFactor = SConfig::GetInstance().m_OCEnable ? SConfig::GetInstance().m_OCFactor : 1.0f;
+	PowerPC::ppcState.downcount = CyclesToDowncount(slicelength);
```

Comment:
I think the OC factor is easier for people to understand?  I don't see how IPC is more correct based on what it's doing.